audbackend/core/backend.py

```diff
@@ -210,13 +217,25 @@ def get_file(
 
         """
         utils.check_path_for_allowed_chars(src_path)
-        if not self._exists(src_path, version):
-            utils.raise_file_not_found_error(src_path, version=version)
 
         dst_path = audeer.safe_path(dst_path)
-        audeer.mkdir(os.path.dirname(dst_path))
-
-        self._get_file(src_path, dst_path, version, verbose)
+        dst_root = os.path.dirname(dst_path)
+
+        audeer.mkdir(dst_root)
+        if (
+                not os.access(dst_root, os.W_OK) or
+                (os.path.exists(dst_path) and not os.access(dst_root, os.W_OK))
+        ):  # pragma: no cover
```

Comment:
@hagenw do you know how we can exclude code from code coverage on Windows only?

Reply:
One way would be to create a `.coveragerc` on the fly in the runner on Windows and add the following content:

```config
[report]
exclude_lines =
    pragma: no cover
    pragma: no Windows cover
```

and change the line here to

```suggestion
        ):  # pragma: no Windows cover
```

Reply:
Thanks, I now switched to custom configuration files for each OS and applied your suggestion.